.eslintrc.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
     "@typescript-eslint/no-empty-function": 0,
     "@typescript-eslint/no-empty-interface": 0,
     "@typescript-eslint/no-non-null-assertion": 0,
-
+    artsyPaletteCanonicalImport: 2,
```

Comment:
Does this come with an autofixer?

Reply:
Yes:

https://github.com/artsy/force/pull/6208/files#diff-47ff37c01755fe96a9acaf746d0d7bf3R117